src/health/healthz.ts

```diff
@@ -14,9 +14,10 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-import config from "../config";
 import * as http from "http";
 import { LogService } from "matrix-bot-sdk";
+// allowed to use the global configuration since there should only be one of these anyways.
```

Comment:
True but it took me a little time to understand what you meant by "one of these". Maybe "`Healthz` is expected to be a singleton, even in presence of multiple instances of `Mjolnir`"? Is that true, btw?

Reply:
Yes, it's not really clear to me how the `Healthz` monitoring is useful since if you make it past startup then there's no way for it to become unhealthy. Either way I changed this again so that Healthz is only used from `src/index.ts`. This is because I don't know who is depending on the `Healthz` monitoring and if we allowed the monitoring to be configurable per Mjolnir, we would end up starting the web server much later (from the perspective of someone using Mjolnir via index.ts (ie everyone))